modules/managers/galasa-managers-parent/galasa-managers-cicsts-parent/dev.galasa.cicsts.manager/src/main/java/dev/galasa/cicsts/internal/CicstsDefaultLogonProvider.java

```diff
@@ -134,7 +134,7 @@ private void waitForLoggedOnText(ICicsTerminal cicsTerminal) throws CicstsManage
 
         String[] pass = { "Sign-on is complete" };
         String[] fail = { 
-            "Your password has expired. Please type your new password.",
+            "Your password has expired. Type your new password.",
```

Comment:
This message might come from the server since the terminal waits for it to appear. I think this should be changed back to avoid any issues for tests using this manager.

Reply:
Reverted